hedera-dependency-versions/build.gradle.kts

```diff
@@ -37,6 +37,9 @@ val tuweniVersion = "2.3.1"
 
 dependencies {
     api(enforcedPlatform("io.netty:netty-bom:$nettyVersion"))
+
+    // forward logging from modules using SLF4J (e.g. 'org.hyperledger.besu.evm') to Log4J
+    runtime(javaModuleDependencies.gav("org.apache.logging.log4j.slf4j"))
```

Comment:
Central place to define additional dependencies we want for all "runtimes" (test execution / application execution) but do not want to publish. If our modules are used somewhere else, the consumer should be able to decide on logger implementation(s). (Ideally, log4jcore could also move here such that our code only depends on log4j-api.)